autosubmit/platforms/ecplatform.py

```diff
@@ -37,18 +37,6 @@ class EcPlatform(ParamikoPlatform):
     :type scheduler: str (pbs, loadleveler)
     """
 
-    def parse_Alljobs_output(self, output, job_id):
-        pass
-
-    def parse_queue_reason(self, output, job_id):
-        pass
-
-    def get_checkAlljobs_cmd(self, jobs_id):
-        pass
-
-    def submit_Script(self, hold=False):
-        pass
-
```

Comment:
I believe the idea here is exactly to have these functions so that calling them won't call the parent class. If `ECPlatform` doesn't have `parse_Alljobs_output`, for instance, I think if `ecplatform_instance.parse_Alljobs_output()` is called, then it would call `parse_Alljobs_output` from the `ParamikoPlatform`. And instead of `pass`, that function would raise a `NotImplementedError`, which could cause AS to crash.

Can you confirm that, @VindeeR ? If that's really the case, then I believe we don't want to remove these functions...

Reply:
It is how it was designed :shrug: I think it's safer to leave as it was before.